drivers/usb/device/usb_dc_mcux.c

```diff
@@ -470,12 +467,14 @@ int usb_dc_ep_disable(const uint8_t ep)
 		return -EINVAL;
 	}
 
-	status = dev_state.dev_struct.controllerInterface->deviceCancel(
-						  dev_state.dev_struct.controllerHandle,
-						  ep);
-	if (kStatus_USB_Success != status) {
-		LOG_ERR("Failed to disable ep 0x%02x", ep);
-		return -EIO;
+	if (dev_state.dev_struct.controllerHandle != NULL) {
```

Comment:
just wondering why do we check for this here now?

Reply:
The test is calling `usb_dc_ep_disable` before the endpoint is initialized which is a valid scenario.